data/comment-from-api.data.go

```diff
@@ -0,0 +1,39 @@
+package data
+
+import(
+	"github.com/nhaancs/GrabGoTrainingWeek5Assignment/core/entity"
+	"github.com/nhaancs/GrabGoTrainingWeek5Assignment/data/dataformatter"
+	"net/http"
+	"io/ioutil"
+)
+
+// CommentData data
+type CommentData struct {
+	DataFormatter dataformatter.DataFormatter
+}
+
+// NewCommentData func
+func NewCommentData(dataformatter dataformatter.DataFormatter) *CommentData {
+	return &CommentData{
+		DataFormatter: dataformatter,
+	}
+}
+
+// GetComments get all comments
+func (data *CommentData) GetComments() ([]entity.Comment, error) {
```

Comment:
Nitpick: this function do 2 things, get data and render it, should have separate into 2 smaller functions